src/common/entity/color/climate_color.ts

```diff
@@ -1,3 +1,13 @@
+import { HvacAction } from "../../../data/climate";
+
+export const CLIMATE_HVAC_ACTION_COLORS: Record<HvacAction, string> = {
+  cooling: "var(--rgb-state-climate-cool-color)",
+  drying: "var(--rgb-state-climate-dry-color)",
+  heating: "var(--rgb-state-climate-heat-color)",
+  idle: "var(--rgb-state-climate-idle-color)",
+  off: "var(--rgb-state-climate-off-color)",
+};
```

Comment:
What happens to the existing CSS vars such as `--state-climate-auto-color`?

And looking at the current vars there are more, so I would assume we also need them here now?

Reply:
Action can only have this values.
State is managed with `stateColor`.

Reply:
I think all these variables can be removed.

Reply:
Thermostat card uses the `--state-climate-{state}-color` variables. And probably other custom card. We can keep it for now.

Reply:
Hi @piitaya   If I update HA straight to 2022.12.5, do the thermostat state colour will work as before?

Reply:
I have my answer...  Yes, everything back to normal... 😊

Thx